keras_rs/src/metrics/n_dcg.py

```diff
@@ -0,0 +1,154 @@
+from typing import Any, Callable, Optional
+
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.api_export import keras_rs_export
+from keras_rs.src.metrics.ranking_metric import RankingMetric
+from keras_rs.src.metrics.ranking_metric import (
+    ranking_metric_subclass_doc_string,
+)
+from keras_rs.src.utils.ranking_metrics_utils import compute_dcg
+from keras_rs.src.utils.ranking_metrics_utils import default_gain_fn
+from keras_rs.src.utils.ranking_metrics_utils import default_rank_discount_fn
+from keras_rs.src.utils.ranking_metrics_utils import get_list_weights
+from keras_rs.src.utils.ranking_metrics_utils import sort_by_scores
+
+
+@keras_rs_export("keras_rs.metrics.nDCG")
+class nDCG(RankingMetric):
```

Comment:
The convention is to always use an upper case letter for the first letter of a class, so it should be `NDCG`, even if it's written as `nDCG` in papers.

Also the file should be `ndcg.py`.

I don't actually understand why it's written `nDCG` since all the letters form an acronym anyway.